src/ontology/edits/oeo-model.omn

```diff
@@ -1047,13 +1047,42 @@ Class: OEO_00010255
         <http://purl.obolibrary.org/obo/IAO_0000118> "NDC",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/947
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1151",
-        rdfs:label "nationally determined contribution"@de
+        rdfs:label "nationally determined contribution"
     
     SubClassOf: 
         OEO_00020093,
         <http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00000199
     
     
+Class: OEO_00010260
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Parameterisation is the process of translating assumptions into something the model can use.",
```

Comment:
Minor suggestion for change: _Parameterisation is the process of translating assumptions into input data a model can use._

Reply:
I like this suggestion as "something" is rather unspecific. However, I think we can skip the word "quantitative" as `exogenous data` is already by definition quantitativ: _Endogenous data is a data item whose quantity value is determined by a model._

We might axiomatise this by adding an axiom `'endogenous data' 'is about' some 'quantity value'` (and likewise: `'endogenous data' 'is about' some 'quantity value'`.

Reply:
sounds good.